src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,65 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.*;
+/**
+ * Represents block number of a person's address
+ * @author langyanbin0314
```

Comment:
You can omit the `@author` tag as we expect multiple developers to edit the code.